src/main/java/com/softserve/kickscootertrip/converters/ListDtoToEntity.java

```diff
@@ -0,0 +1,22 @@
+package com.softserve.kickscootertrip.converters;
+
+import com.softserve.kickscootertrip.model.dto.TripDto;
+import com.softserve.kickscootertrip.model.entity.TripEntity;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class ListDtoToEntity implements Converter<List<TripDto>, List<TripEntity>> {
```

Comment:
The same point about the class name as above

Reply:
you shouldn't write a converter for lists, what if you need to convert `Set` or `Map`? Just use an already created converter